### PR TITLE
Remove unneeded blank lines from cxx files

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: main/automation/source/server/recorder.cxx [hunk 1]
```diff
@@ -586,7 +586,7 @@ IMPL_LINK( MacroRecorder, EventListener, VclSimpleEvent*, pEvent )
 					if ( pTB->GetUniqueOrHelpId() != nUId )	// Also Button auf der ToolBox gefunden
 					{
 						if ( nParams == PARAM_NONE )
-						{			// Wir f�lschen einen Parameter
+						{			// Wir f�lschen einen Parameter
```
Comment (jbampton) [inline main/automation/source/server/recorder.cxx]:
```suggestion
						{			// Wir fälschen einen Parameter
```

File: main/automation/source/server/recorder.cxx
Comment (Pilot-Pirx) [inline main/automation/source/server/recorder.cxx:1]: Yes, still old German comments from the time where no Unicode was used... ;-)

Comment (jbampton) [reply main/automation/source/server/recorder.cxx:1]: Some of these German words have found their way into our codespell ignored words list in #191

Comment (Pilot-Pirx) [reply main/automation/source/server/recorder.cxx:1]: Often German Umlauts were replaced:
ä -> ae
ö -> oe
ü -> ue
ß -> ss
I also saw:
ä -> "a
ö -> "o
ü -> "u
Basically, we should translate these comments finally... ;-)

**Review by Pilot-Pirx — APPROVED** [review]

**Pull request merged**